src/main/flight/feedforward.c

```diff
@@ -136,10 +136,12 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
             // does not attenuate the basic feedforward amount, but this is small anyway at centre due to expo
             setpointAcceleration *= jitterAttenuator;
 
-            if (absSetpointPercent > 0.95f && absSetpointSpeed < 3.0f * absPrevSetpointSpeed) {
-                // approaching max stick position so zero out feedforward to minimise overshoot
-                setpointSpeed = 0.0f;
-                setpointAcceleration = 0.0f;
+            if (!FLIGHT_MODE(ANGLE_MODE)) {
```

Comment:
Is it intended not to include `FLIGHT_MODE(HORIZON_MODE)` here?

Reply:
Yes. We still want max rate limit on FF in horizon mode.

Reply:
I'm not sure.  In Horizon mode the PID loop feedforward should be calculated normally, as in acro, from rates * stick position.  The input should not come from angle setpoint.